### ISSUE TITLE
Not deploying to AWS

### ISSUE THREAD

**Issue by thiagoramos23** [nficano/python-lambda#35]
Nice Lib. So much simpler than Zappa. It's really what I need, but I am having hard times to deploy. It got stuck when calling lambda deploy.

> Found credentials in shared credentials file: ~/.aws/credentials
> Starting new HTTPS connection (1): lambda.us-west-2.amazonaws.com
> Creating your new Lambda function
> Starting new HTTPS connection (1): sts.amazonaws.com
> Starting new HTTPS connection (1): lambda.us-west-2.amazonaws.com
> Starting new HTTPS connection (2): lambda.us-west-2.amazonaws.com
> Starting new HTTPS connection (3): lambda.us-west-2.amazonaws.com

**Comment by RedOakMark** [comment]
This probably has to do with the size of your deployment package @thiagoramos23 - I believe there is a 50MB limit and as soon as I topped that number, I got the same error.  

This was in my `dist` folder.  The 39MB package worked fine, but the 98MB package failed with this error `botocore.vendored.requests.exceptions.ConnectionError: ('Connection aborted.', error(32, 'Broken pipe'))`. 

![image](https://user-images.githubusercontent.com/15692808/27348933-b1417310-55c3-11e7-8df7-6823cbbbd357.png)
